File: src/Montr.Tendr/Models/Invitation.cs
using System;

namespace Montr.Tendr.Models
{
	public class Invitation
	{
		public Guid Uid { get; set; }

		public string StatusCode { get; set; }

		public Guid CounterpartyUid  { get; set; }

		public string Email { get; set; }
	}
}

File: src/ui/modules/montr-tendr/models/invitation.ts
import { IIndexer, Guid } from "@montr-core/models";

export interface Invitation extends IIndexer {
	uid?: Guid;
	eventUid?: Guid,
	statusCode?: string;
	counterpartyUid?: Guid;
}

export interface InvitationListItem extends IIndexer {
	uid?: Guid;
	eventUid?: Guid,
	statusCode?: string;
	counterpartyUid?: Guid;
	counterpartyName?: string;
	email?: string;
}
